etc/profile-a-l/electron.profile

```diff
@@ -16,7 +16,7 @@ include disable-xdg.inc
 
 whitelist ${DOWNLOADS}
 whitelist ${HOME}/.config/Electron
-whitelist ${HOME}/.config/electron-flag*.conf
+whitelist ${HOME}/.config/electron[0-9]*-flag*.conf
```

Comment:
What is the format for electron-flag files? You edit looks like you maybe tried to use a regexp. Firejail uses `glob(3)` for globbing, the format is described in `glob(7)`.

This will match `electron` + one digit + anything (zero or more of anything) + `-flag` + anything + `.conf`.

Reply:
It is `electron<version>-flags.conf`, i wasn't sure if maybe some older version also used `-flag.conf` so i kept the `*` there.
Yes, that was the intended pattern. (Since it needs to match both `electron-flags and `electron16-flags` for example, because 1. some versions of electron do use the `electron-flags` file IIRC and 2. the `electron<version>-flags` files are often symlinked to this one common file for convenience).